doc/examples/edges/plot_circular_elliptical_hough_transform.py

```diff
@@ -82,10 +82,10 @@
 # -------------------
 #
 # The algorithm takes two different points belonging to the ellipse. It
-# assumes that it is the main axis. A loop on all the other points determines
-# how much an ellipse passes to them. A good match corresponds to high
-# accumulator values.
-#
+# assumes that it is the major axis. A loop on all the other points determines
+# potential minor axis lengths of candidate ellipses. Ellipses are added to the
```

Comment:
```suggestion
# the minor axis length for candidate ellipses. The latter are included in the
```
I just thought that:

* term 'potential' is redundant with 'candidate;'
* plural is unnecessary since each ellipse has a (its own) minor axis length (I would think there's no possible confusion);
* I guess 'added to' is fine, it echoes the loop/accumulator, but 'included in' makes it clear that it's found in the 'final' result(?).